### PR TITLE
initial setup for wagtail and phoenix

### PR DESCRIPTION
**Pull request by samhstn**
Simply running the command:

```bash
mix phoenix.new app
```

And adding content to the readme

### CODE REVIEW

**Comment by nelsonic** [comment]
Hi Sam, as per the digram in #9 we are keeping the CMS _separate_ from the App. 
Please transfer any CMS related code to https://github.com/LDMW/cms thanks. ❤️  ✅

**Comment by shouston3** [comment]
I'll change that now

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
-# app
\ No newline at end of file
+# app
+
+Built using the _PETE_ stack [`Phoenix`, `Elixir`](http://www.phoenixframework.org/), [`Tachyons`](http://tachyons.io/), and [`Elm`](http://elm-lang.org/)
+
+As well as the [`Wagtail`](https://wagtail.io/) CMS
```
Comment (nelsonic) [inline README.md]: could it be worth including the diagram #9 here?

File: README.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+Clone the repository and cd into it (make sure you don't have another project called `app`):
+
+```bash
+git clone https://github.com/ldmw/app.git && cd app
+```
+
+Ensure to have the following environment variables in your `$PATH`
+
+```bash
```
Comment (nelsonic) [inline README.md]: don't know if we _still need_ these Environment Variables.

File: README.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+
+```elixir
+mix phoenix.server
+```
+
+(For information on any of these commands run `mix help <command>`)
+
+The project should now be running at `https://localhost:4000`
+
+#### CMS setup:
```
Comment (nelsonic) [inline README.md]: these details should probably go in the `cms` project readme...

**Review by nelsonic — APPROVED** [review]
@shouston3 fairly standard setup. (_looks good_)
thanks for adding detail to the docs/README. 👍 
We _could_ remove the Wagtail/CMS-related docs/instructions in a future ("_tidy up_") PR and add the diagram #9 for clarity for non-technical people. 
But this is a great start! thanks! 🎉

**Pull request merged**